tests/filecheck/dialects/vector/invalid.mlir

```diff
@@ -0,0 +1,63 @@
+// RUN: xdsl-opt %s --split-input-file --verify-diagnostics | filecheck %s
+
+func.func @insert_vector_type(%a: f32, %b: vector<4x8x16xf32>) {
+  // CHECK: Expected position attribute rank (6) to match dest vector rank (3).
+  %1 = vector.insert %a, %b[3, 3, 3, 3, 3, 3] : f32 into vector<4x8x16xf32>
+  func.return
+}
+
+// -----
+
+func.func @insert_vector_type(%a: vector<4xf32>, %b: vector<4x8x16xf32>) {
+  // CHECK: Expected position attribute rank + source rank (2) to match dest vector rank (3).
```

Comment:
I got confused by original phrasing, since it's close to source rank I assumed it was not the sum that is 2
```suggestion
  // CHECK: Expected position attribute rank + source rank (1+1) to match dest vector rank (3).
```